scripts/create-model.py

```diff
@@ -6,9 +6,10 @@
 import yaml
 import os
 
-from ed_object_models.conversion_sdf import main as convert_main
+import rospkg
```

Comment:
In general, this is IMHO not a very nice way to create a model. It would be much nicer to be able to add stuff to an Entity and to be able to dump an entity to file. However, this is out of the scope of this MR